rust-runtime/aws-smithy-http-tower/src/lib.rs

```diff
@@ -102,7 +103,10 @@ mod tests {
         });
 
         let mut svc = ServiceBuilder::new()
-            .layer(ParseResponseLayer::<TestParseResponse, ()>::new())
+            .layer(ParseResponseLayer::<
+                TestParseResponse,
```

Comment:
Consider calling this "TestResponseHandler" or, instead, changing the verbiage for "response handler" to "response parser"

Reply:
I think the name is actually fine considering it implements `ParseStrictResponse`. I think `handler` might actually be the out of place name that should potentially get fixed in other places.